### ISSUE TITLE
Mandlebrot example fails to run

### ISSUE THREAD

**Issue by Laevos** [thiagopnts/groot#8]
On Nightly Rust (rustc 1.5.0-nightly (6108e8c3e 2015-09-28)), the following error occurs when trying to run the Mandlebrot example:

```
Running `target/debug/groot mandelbrot.groot`
thread '<main>' panicked at 'arithmetic operation overflowed', src/interpreter.rs:52                                      
AAAAAAAAAAAAAAAABBBBBBBBAn unknown error occurred
```

(Note, this is using the version of my fork of this code, see pull request #7 )

**Comment by webmaven** [comment]
Is this still an issue?

**Comment by Laevos** [comment]
No, this was fixed when PR #7 was merged.
